models/oneinch/fantom/oneinch_fantom_calls_transfers.sql

```diff
@@ -0,0 +1,126 @@
+{{ config( 
+    alias = alias('fantom_calls_transfers'),
```

Comment:
please supply a `schema` property in addition to alias
`schema = '1inch_fantom',`
then alias can simplify:
`alias = alias('calls_transfers'),`

Reply:
could it be just in current `oneinch` schema? (actually i've tried to pass schema and removed, thought that was causing the problem lol)

it'd be like
`oneinch.fantom_calls_transfers`
`oneinch.ethereum_calls_transfer`
`...`

Reply:
we have a pattern spellbook-wide to write schemas in this format:
`<project/sector_name>_<version>_<blockchain>.<table_alias>`
where:
- version is only if applicable
- blockchain is included when we start to build cross-chain spells, so we can keep a clean lineage per blockchain and then into cross-chain

after you build all the blockchain-specific schemas and spells, the cross-chain will just be `1inch`

it also follows our directory path naming standard

Reply:
got you, thanks for the clear explanation!